lib/cli.mjs

```diff
@@ -383,6 +383,8 @@ const meowOptions = {
  * @returns {Promise<any>}
  */
 export default async function main(argv) {
+	process.exitCode = 0;
```

Comment:
Thanks for the suggestion! Done here: 73f66818a540d9a06dc58bb53d76d22a937454c6

I am not sure if we want to keep this line, because as @Mouvedia suggested above, we could also resetting `process.exitCode` in tests. Please see my [comment](https://github.com/stylelint/stylelint/pull/7119#discussion_r1289122158) above and let me know what you think. Thanks!